src/sentry/api/issue_search.py

```diff
@@ -192,6 +192,22 @@ def convert_category_value(
     return results
 
 
+def convert_priority_value(
+    value: Iterable[str],
+    projects: Sequence[Project],
+    user: User,
+    environments: Sequence[Environment] | None,
+) -> list[int]:
+    """Convert a value like 'high' or 'medium' to the Priority value for issue lookup"""
+    results: list[int] = []
+    for priority in value:
+        priority_value = getattr(PriorityLevel, priority.upper(), None)
```

Comment:
Is it worth using this mapping instead? (might need a rename though) https://github.com/getsentry/sentry/blob/2ffd8258b9a242b18b7682ee85003d34fb59a67b/src/sentry/issues/priority.py#L25-L29